Lib/distutils/tests/test_ccompiler.py

```diff
@@ -24,6 +24,28 @@ def library_option(self, lib):
 
 class CCompilerTestCase(support.EnvironGuard, unittest.TestCase):
 
+    def test_set_executables_unicode(self):
```

Comment:
Please rename to test_set_executable(), since you test 3 types, not only Unicode.